weave

```diff
@@ -1066,10 +1081,7 @@ http_call_unix() {
 
 call_weave() {
     check_running $CONTAINER_NAME || return 1
-    if ! http_call $HTTP_ADDR "$@" ; then
-        echo "Call to $CONTAINER_NAME failed." >&2
-        return 1
-    fi
+    http_call $HTTP_ADDR "$@"
```

Comment:
Is it easy to generate that failure?  We have just checked that `weave` is running, which is the most obvious case.